src/test/FuzzerImpl.cpp

```diff
@@ -1193,6 +1215,40 @@ TransactionFuzzer::initialize()
         ltx.commit();
     }
 
+    {
+        LedgerTxn ltx(ltxOuter);
+
+        xdr::xvector<Operation> ops;
+
+        for (auto const& param : claimableBalanceParameters)
+        {
+            PublicKey claimantKey;
+            FuzzUtils::setShortKey(claimantKey, param.mClaimant);
+
+            PublicKey senderKey;
+            FuzzUtils::setShortKey(senderKey, param.mSender);
+
+            ClaimPredicate predicate;
+            predicate.type(CLAIM_PREDICATE_UNCONDITIONAL);
+            Claimant claimant;
+            claimant.v0().predicate = predicate;
+            claimant.v0().destination = claimantKey;
```

Comment:
Nitpick: no need to define `claimantKey` earlier, you can just use `FuzzUtils::setShortKey(claimant.v0().destination, ...)`

Reply:
Done.